analysis/SimulationDataset.py

```diff
@@ -78,7 +78,6 @@ def __init__( self, InputPath, TotalDecays, EnergyMin, EnergyMax, CoincidenceWin
       self.unusedEvents.append( i )
 
   def SampleOneEvent( self, EnergyResolution=0.0, TimeResolution=0.0 ):
-
```

Comment:
Try and avoid arbitrary changes to whitespace

Reply:
Fixed